MovieTracker App/Scenes/TV Series/Component/TvSeriesListItem.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  TvSeriesListItem.swift
+//  MovieTracker App
+//
+//  Created by Macbook Air 13 on 07.01.24.
+//
+
+import SwiftUI
+
+struct TvSeriesListItem: View {
+    
+    // MARK: - Properties
+    @Binding var tvSeriesItem: TvSeries
+    var posterBaseURL = "https://image.tmdb.org/t/p/original"
+    
```

Comment:
აქ private

Reply:
აა მაგას ვერ შევცვლიდი იმიტო რო მერე სხვა ფაილში მიწერს: 'TvSeriesListItem' initializer is inaccessible due to 'private' protection level

Reply:
რატომღაც მეგონა, რომ მარტო ამ ფაილში იყენებდი :დ